tests/foreman/api/test_computeresource_azurerm.py

```diff
@@ -164,7 +164,7 @@ def class_setup(self, request, module_domain, module_azurerm_cr, module_azurerm_
         request.cls.premium_os_disk = AZURERM_PREMIUM_OS_Disk
         request.cls.platform = AZURERM_PLATFORM_DEFAULT
         request.cls.vm_size = AZURERM_VM_SIZE_DEFAULT
-        request.cls.hostname = gen_string('alpha')
+        request.cls.hostname = f"test_{gen_string('alpha')}"
```

Comment:
Please switch these to single-quote strings here and through the other changed lines

Reply:
```suggestion
        request.cls.hostname = f'test_{gen_string('alpha')}'
```

This wont work as there is already single quote inside !

Reply:
There are 4 quotes on one line !

Reply:
you can flip them, so it looks like 
```suggestion
        request.cls.hostname = f'test_{gen_string("alpha")}'
```

Reply:
Not sure whats the advantage it has but as per python standards, we should wrap single quotes with double and not vice-versa.

Still made the changes are are ready for your ACK !